src/libraries/System.Formats.Tar/tests/TarWriter/TarWriter.File.Base.cs

```diff
@@ -0,0 +1,70 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using Xunit;
+
+namespace System.Formats.Tar.Tests
+{
+    public partial class TarWriter_File_Base : TarTestsBase
+    {
+        // TestTarFormat, string testCaseName, CompressionMethod, bool copyData
+        public static IEnumerable<object[]> WriteEntry_CopyArchive_Data()
+        {
+            foreach (CompressionMethod compressionMethod in Enum.GetValues<CompressionMethod>())
+            {
+                foreach (bool copyData in new object[] { false, true })
+                {
+                    foreach (object[] testCaseName in GetV7TestCaseNames())
+                    {
+                        if ((string)testCaseName[0] == "folder_file_utf8") // The legacy name field is stored in ASCII in the V7 format
+                        {
+                            continue;
+                        }
+
+                        yield return new object[] { TestTarFormat.v7, testCaseName[0], compressionMethod, copyData };
+                    }
+
+                    foreach (object[] testCaseNameObj in GetUstarTestCaseNames())
+                    {
+                        string testcaseName = (string)testCaseNameObj[0];
+                        if (testcaseName is "folder_file_utf8" // The legacy name field is stored in ASCII in the Ustar format
+                                         or "longpath_splitable_under255" // Folder stored under 'unarchived' runtime-assets due to NuGet not allowing very long paths
```

Comment:
Maybe too late to change if it's an asset but "splittable". No need to reset for this

Reply:
I _knew_ that word looked weird. That is my typo. But yes, it's an asset.